.circleci/config.yml

```diff
@@ -55,6 +55,18 @@ jobs:
       - run:
           name: Check style differences
           command: git --no-pager diff --exit-code
+  Registers:
+    <<: *docker_config
+    environment:
+      CLANG_FORMAT: clang-format-5.0
+    steps:
+      - checkout
+      - run:
+          name: Generate register descriptors
+          command: ./Support/Scripts/generate-register-descriptors.sh
+      - run:
```

Comment:
You can make this into a separate step, like we do for `build_step`, `configure_step`, etc, so you don't have to duplicate the code that's already in the `Style` check.